devito/petsc/solve.py

```diff
@@ -105,3 +115,52 @@ def _(expr, target):
 @remove_target.register(Derivative)
 def _(expr, target):
     return 0 if expr.has(target) else expr
+
+
+def centre_stencil(expr, target):
```

Comment:
This function seems redundant?

Reply:
I just haven't used it yet but I will need the functionality

Reply:
I guess it will do more stuff later?

Reply:
The function won't do more, it will be used elsewhere

Reply:
Oh sorry I see what you mean, I removed the `expr.evaluate` so yes it is now redundant